templates/types/streaming/nextjs/app/api/data/route.ts

```diff
@@ -0,0 +1,40 @@
+import { readFile } from "fs/promises";
```

Comment:
seems like this is an easier alternative: https://www.perplexity.ai/search/using-nextjs-how-jI1_WonTRHWG1XG4_BtYzw

Reply:
that's the problem with asking AIs :D